GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -354,6 +356,80 @@ - (void)removeLibraryDataForKey:(nonnull NSString *)key
   });
 }
 
+- (BOOL)hasEventsForTarget:(GDTCORTarget)target {
+  NSString *searchPath = [GDTCORFlatFileStorage pathForTarget:target qosTier:nil mappingID:nil];
+  NSFileManager *fm = [NSFileManager defaultManager];
+  NSDirectoryEnumerator *enumerator = [fm enumeratorAtPath:searchPath];
+  while ([enumerator nextObject]) {
+    if ([enumerator.fileAttributes[NSFileType] isEqual:NSFileTypeDirectory] == NO &&
```

Comment:
Why not test for `NSFileTypeRegular` instead?

Reply:
That's probably a lot smarter.